core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -55,77 +56,132 @@ object ConsumerGroupCommand {
 
     val consumerGroupService = {
       if (opts.useOldConsumer) {
+        println("Note: This will only show information about consumers that use ZooKeeper (not those using the Java consumer API).\n")
         new ZkConsumerGroupService(opts)
       } else {
+        println("Note: This will only show information about consumers that use the Java consumer API (non-ZooKeeper-based consumers).\n")
         new KafkaConsumerGroupService(opts)
       }
     }
 
     try {
       if (opts.options.has(opts.listOpt))
-        consumerGroupService.list()
-      else if (opts.options.has(opts.describeOpt))
-        consumerGroupService.describe()
+        consumerGroupService.listGroups().foreach { printMessage(_) }
+      else if (opts.options.has(opts.describeOpt)) {
+        val describeGroupResult = consumerGroupService.describeGroup()
+        describeGroupResult._2 match {
+          case None =>
+            printError(s"The consumer group '${opts.options.valuesOf(opts.groupOpt).asScala.head}' does not exist.")
+          case Some(assignments) =>
+            if (assignments.isEmpty)
+              if (describeGroupResult._1 == "Empty")
+                printError(s"Consumer group '${opts.options.valuesOf(opts.groupOpt).asScala.head}' has no active member.")
```

Comment:
nitpick: should 'member' be plural? Also, a little surprising we don't have a variable for the groupId. Maybe we could add one above to make these messages a little easier to read.